app/src/main/res/values/strings.xml

```diff
@@ -291,36 +291,36 @@
     <string name="persistent_notification_title">Persistent notification</string>
     <string name="persistent_notification_summary">Add a notification when the database is open</string>
     <string name="advanced_unlock">Advanced unlock</string>
-    <string name="advanced_unlock_explanation_summary">Use advanced unlocking to open a database more easily</string>
+    <string name="advanced_unlock_explanation_summary">Use automated database unlocking</string>
     <string name="biometric_unlock_enable_title">Biometric unlocking</string>
     <string name="biometric_unlock_enable_summary">Lets you scan your biometric to open the database</string>
-    <string name="biometric_auto_open_prompt_title">Auto open biometric prompt</string>
-    <string name="biometric_auto_open_prompt_summary">Automatically open biometric prompt when a biometric key is defined for a database</string>
+    <string name="biometric_auto_open_prompt_title">Auto-open biometric prompt</string>
+    <string name="biometric_auto_open_prompt_summary">Ask for biometric if the database is set up to require it</string>
```

Comment:
Not the same meaning

Reply:
How so?

Reply:
Could this be explained? My understanding was/is that my version means the same…